client/components/Coverages/CoverageEditor/index.jsx

```diff
@@ -69,11 +70,10 @@ export const CoverageEditor = ({
             callback: duplicateActions,
         }];
 
-        if (value.coverage_id) {
+        if (isExistingItem(value, 'coverage_id')) {
```

Comment:
This is better off placed in the `canCancelCoverage` utility function.
On the same note, a new utility function for `canAddToWorkflow` should be created and used as well.

This keeps the item checking vs available actions in the one place.